rust/agents/scraper/migration/src/m20230309_000005_create_table_message.rs

```diff
@@ -135,7 +135,7 @@ impl MigrationTrait for Migration {
                 "origin_tx"."{tx_egp}" AS "origin_tx_effective_gas_price",
                 "origin_tx"."{tx_nonce}" AS "origin_tx_nonce",
                 "origin_tx"."{tx_sender}" AS "origin_tx_sender",
-                "origin_tx"."{tx_receipient}" AS "origin_tx_recipient",
+                "origin_tx"."{tx_recipient}" AS "origin_tx_recipient",
```

Comment:
We need to please revert the changes to this database migration file or it will break the Hyperlane explorer